experimental/auth-logic/endorsement-release/README.md

```diff
@@ -0,0 +1,22 @@
+This directory includes an executable binary for using authorization logic to decide if endorsement files can be produced.
+
+It takes as input 
+
+- One or more authorization logic files. These include a policy for releasing endorsement files written by the product team and zero or more other authorization logic files that might implement libraries.
+- A provenance file for building the binary to be released.
+
+It generates as output:
+ - An authorization logic file that extends the input policies with the outputs of the provenance file wrapper and the unix time wrapper.
+
+The build rule endorsement_release_oak_functions runs this binary with an example. The input files for this example are:
+ - `experimental/auth-logic/endorsement-release/input_policy_examples/oak_endorsement_policy.auth_logic` -- the policy set by the Oak functions loader team for deciding when to release a binary
+ - `experimental/auth-logic/endorsement-release/input_policy_examples/github_actions_policy.auth_logic`
+
+The real output is generated in:
+ - `bazel-bin/experimental/auth-logic/endorsement-release/oak_endorsement_release_output_policy.auth_logic`
+and a copy of this is stored in:
+ - `experimental/auth-logic/endorsement-release/output_policy_examples/oak_endorsement_release_output_policy.auth_logic`
```

Comment:
Is there anything that automates / checks that this copy is correct?

Reply:
No, but I think now is not the time to do this, because now that I think you're on board with how this works, I will probably adjust the example policy somewhat. Here's an issue: #103

Reply:
I guess "copying" is not stalled or affected by changes to the policies (even though "checking" would be). In any case, this seems like a task for a separate PR to me.